### PR TITLE
Added .webm support to .mkv support: https://github.com/google/spatial-media/issues/4 .

### PR DESCRIPTION
**Pull request by suderman-google**

### CODE REVIEW

**Comment by suderman-google** [comment]
https://github.com/google/spatial-media/issues/4

File: 360-Videos-Metadata/360VideosMetadata.py [hunk 1]
```diff
@@ -768,11 +768,12 @@ def PrintMetadata(src):
 
     print "Processing: ", infile, "\n"
 
-    if ".mkv" in infile:
+    if (os.path.splitext(infile)[1].lower() == ".webm" or
+       os.path.splitext(infile)[1].lower() == ".mkv"):
```
Comment (ascherkus) [inline 360-Videos-Metadata/360VideosMetadata.py]: should be indented by one more space

File: 360-Videos-Metadata/360VideosMetadata.py [hunk 2]
```diff
@@ -797,11 +798,12 @@ def InjectMetadata(src, dest):
 
     print "Processing: ", infile, "\n"
 
-    if ".mkv" in infile:
+    if (os.path.splitext(infile)[1].lower() == ".webm" or
```
Comment (ascherkus) [inline 360-Videos-Metadata/360VideosMetadata.py]: ditto

File: 360-Videos-Metadata/360VideosMetadata.py (hunk 1 above)
Comment (suderman-google) [inline 360-Videos-Metadata/360VideosMetadata.py]: Causes an error for pep8 because it doesn't distinguish from the next line.

Comment (ascherkus) [inline 360-Videos-Metadata/360VideosMetadata.py]: ah riiiighhhtt ... for future situations like that I'd indent next line by 4 from the (

(but still I think the "in [ ... ]" is the way to go here)

**Pull request merged**